README.md

```diff
@@ -134,11 +134,39 @@ services:
     - ...
 
 ```
+### Start with an appliance backup
+
+To start with the appliance backup files (See [here](https://github.com/mekomsolutions/appliance-deployment/blob/main/README.md#backup-profile) for more details on how to get the backup files), follow these steps:
+
+1. Unzip the backup file and rename PostgreSQL database files to:
+  - OpenELIS : `clinlims.tar`
+  - Odoo : `odoo.tar`
+2. Move PostgreSQL database files to [./sqls/postgresql/restore](./sqls/postgresql/restore) folder
+3. For OpenMRS database please folow the steps [here](#start-with-a-custom-mysql-dump)
+4. Unzip the `filestore.zip` file and set the variables in `.env` file as following:
+   - Odoo:
+     - ODOO_FILESTORE=`<filestore-path>/odoo`
+   - OpenMRS:
+     - OPENMRS_LUCENE_PATH=`<filestore-path>/openmrs/lucene`
+     - OPENMRS_ACTIVEMQ_PATH=`<filestore-path>/openmrs/activemq-data`
+     - OPENMRS_CONFIG_CHECKSUMS_PATH=`<filestore-path>/openmrs/configuration_checksums`
+
+5. Start PostgreSQL:
+
+```
+docker-compose [-p <project-name>] up -d postgresql
+```
+
+6. Start the restore service
+
+```
+docker-compose [-p <project-name>] -f postgres_restore.yml up
+```
+
```

Comment:
Should we then turn `postgresql` service down after restore is complete?

Reply:
Should we turn off anyway to make it in "down" state for all the services?

Reply:
>Should we turn off anyway to make it in "down" state for all the services?

I'm not sure I understand this remark.